CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 Changes by Version
 ==================
 
-2.2.0 (unreleased)
+2.1.2 (2016-02-27)
 -------------------
 
-- No changes yet
+- Fix leaky bucket bug
```

Comment:
it's useful to link to the PR, and as Prithvi mentioned the PR should have a description of the bug